Description: Add info dircategory & fix direntry
 The fdutils.texi file was missing the dircategory declaration, and
 the direntry declaration was wrong. This patch sets dircategory to
 "Utilities" and fixes direntry.
Author: Matteo Cypriani <mcy@lm7.fr>
Last-Update: 2010-05-12
--- a/doc/fdutils.texi
+++ b/doc/fdutils.texi
@@ -21,9 +21,10 @@
 
 @ifinfo
 @format
-START-INFO-DIR-ENTRY
+@dircategory Utilities
+@direntry
 * Fdutils: (fdutils).        Fdutils: Linux' floppy utilities
-END-INFO-DIR-ENTRY
+@end direntry
 @end format
 @end ifinfo
 
